com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Requests.java

```diff
@@ -72,6 +78,7 @@
         public boolean stopOnEntry;
         public boolean noDebug = false;
         public CONSOLE console = CONSOLE.internalConsole;
+        public CliStyle shortenCommandLine = CliStyle.none;
```

Comment:
better naming for `shortenCommandLine` , maybe simply `cliStyle`. 

Because I see statement ` if (launchArguments.shortenCommandLine == CliStyle.argfile)` in other files, it confuses me that a `shortenCommandLine` equals to a kind of `CliStyle`